src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -689,6 +689,12 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('workbench.enableExperiments', "Fetches experiments to run from a Microsoft online service."),
 			'default': true,
 			'tags': ['usesOnlineServices']
+		},
+		'workbench.useGridLayout': {
```

Comment:
@sbatten should this have some "experimental" in its name to make clear its being removed in the future?